earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
No, you can just edit that directly.  Maybe post a link to it in a comment on this PR so we can look at it.